src/main/java/nextstep/ladder/domain/Line.java

```diff
@@ -0,0 +1,44 @@
+package nextstep.ladder.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.IntStream;
+
+public class Line {
+    private final List<Boolean> points = new ArrayList<>();
+
+    public Line(int countOfMember) {
+        IntStream.range(0, countOfMember - 1).forEach(i -> this.addPoint(i, new Random().nextBoolean()));
```

Comment:
```suggestion
        IntStream.range(0, countOfMember - 1)
                .forEach(i -> this.addPoint(i, new Random().nextBoolean()));
```
개행을 적절히 넣는다면 가독성이 좋아져요~!

Reply:
짧은건 개행을 안 넣어도 잘보여서 .이 3개정도면 개행을 넣는 편입니다.